.github/workflows/scripts/detect-all-changes.sh

```
@@ -0,0 +1,175 @@
+
+# Detect what components need to be released based on version changes
+# Usage: ./detect-all-changes.sh
+
+echo "🔍 Auto-detecting version changes across all components..."
+
+# Initialize outputs
+CORE_NEEDS_RELEASE="false"
+FRAMEWORK_NEEDS_RELEASE="false"
+PLUGINS_NEED_RELEASE="false"
+BIFROST_HTTP_NEEDS_RELEASE="false"
+CHANGED_PLUGINS="[]"
```

Shellcheck (0.10.0)

[warning] CHANGED_PLUGINS appears unused. Verify use (or export if used externally).

(SC2034)